README.md

```diff
@@ -30,14 +40,33 @@ Play back HLS with video.js, even where it's not natively supported.
 - [Hosting Considerations](#hosting-considerations)
   - [Testing](#testing)
 - [Release History](#release-history)
+- [Building](#building)
+- [Development Commands](#development-commands)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Installation
+### NPM
+To install `videojs-contrib-hls` with npm run
+
+``` Bash
```

Comment:
your code fences are inconsistent, should be lowercase and without a preceding space.